collect_app/src/main/java/org/odk/collect/android/widgets/AudioController.java

```diff
@@ -103,12 +100,16 @@ void init(Context context, MediaPlayer mediaPlayer, FormEntryPrompt formEntryPro
 
     @OnClick(R.id.fastForwardBtn)
     void fastForwardMedia() {
-        seekTo(mediaPlayer.getCurrentPosition() + SEEK_FORWARD_TIME);
+        seekTo(mediaPlayer.getCurrentPosition() + seekMoveTime());
     }
 
     @OnClick(R.id.fastRewindBtn)
     void rewindMedia() {
-        seekTo(mediaPlayer.getCurrentPosition() - SEEK_BACKWARD_TIME);
+        seekTo(mediaPlayer.getCurrentPosition() - seekMoveTime());
```

Comment:
@grzesiek2010 these calls to the `MediaPlayer` are automatically disabled / show no ill effects if triggered before it's prepared right? Just checking.

Reply:
>triggered before it's prepared

you mean that this line might be called before any sound is loaded (mediaPlayer.getCurrentPosition() would cause an exception)? If so I it's rather not a possible scenario.